integrations/dd-sdk-android-okhttp-otel/src/main/kotlin/com/datadog/android/okhttp/otel/OkHttpExt.kt

```diff
@@ -18,7 +18,9 @@ import okhttp3.Request
  */
 fun Request.Builder.addParentSpan(span: Span): Request.Builder {
     val context = span.spanContext
-    val prioritySampling = if (context.isSampled) PrioritySampling.USER_KEEP.toInt() else PrioritySampling.UNSET.toInt()
+    // because the DatadogTracer is using the old legacy DDTracer for handling the spans around request
```

Comment:
what is the `DatadogTracer`? I cannot find such symbol in the code

Reply:
yes because it should be `DatadogInterceptor` there ;)